manifests/v/VladimirYakovlev/ElectronMail/4.12.4/VladimirYakovlev.ElectronMail.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: VladimirYakovlev.ElectronMail
+PackageVersion: 4.12.4
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/vladimiry/ElectronMail/releases/download/v4.12.4/electron-mail-4.12.4-windows-x64-nsis-installer.exe
+  InstallerSha256: A11DF51E3871F76373958FF84579119FECA43FB45F8263D79CFD04BDF8FAE710
+  InstallerSwitches:
+    Silent: /S

```

Comment:
These switches are not needed; winget-cli makes nullsoft installers run with `/S` in the code.